docs/en/trust.rst

```diff
@@ -2,56 +2,298 @@
 
 .. _trust.rst:
 
-trust.rst
+The Infrastructure of Trust
 +++++++++++++++++++++++++++
 
-[What is it]
-    
-[What it is usefull for]
-    
-[Example]
+The EUDI Wallet Architecture Reference Framework (`EIDAS-ARF`_) defines the Trust Model as a *"collection of rules that ensure the legitimacy of the components and the entities involved in the EUDI Wallet ecosystem."*.
+
+This section defines how the Trust Model is implemented in an infrastructure of Trust based on
+OpenID Connect Federation 1.0 `OIDC-FED`_, where its Federation API is used for the distribution of metadata, raw public keys, metadata policies, X.509 certificates, and their revocation statuses.
+
+The infrastructure of Trust enables the trust assessment mechanism to be applied between the parties defined in the `EIDAS-ARF`_.
+
+..  figure:: ../../images/trust-roles.svg
+    :alt: federation portrain
+    :width: 100%
     
+    The roles of the Federation infrastructure, where a Trust Anchor has one or more Intermediates and Leafs.
```

Comment:
Too general. In this document we should to give a concrete architecture of trust, with the same entity's roles as in the ARF.